setup/_pattoo/db.py

```diff
@@ -357,9 +357,24 @@ def _insert_user():
                 password=data.hashstring(password),
                 first_name='pattoo',
                 last_name='pattoo',
+                user_type=0,
+                change_password=0,
```

Comment:
Change this to a 1. There is no need yet for the `pattoo` user to be privileged.